packages/react/src/drawer-content/DrawerContent.tsx

```diff
@@ -0,0 +1,62 @@
+import * as RadixDrawer from "@radix-ui/react-dialog";
+import { forwardRef } from "react";
+
+import { AnimatePresence } from "../animate-presence";
+import { Backdrop } from "../backdrop";
+import { type BoxProps } from "../box";
+import { useDrawerContext } from "../drawer-context";
+import { Paper } from "../paper";
+import { Transition } from "../transition";
+import * as styles from "./DrawerContent.css";
+
+type DrawerContentProps = BoxProps<
+  typeof RadixDrawer.Content,
+  NonNullable<styles.DrawerVariants>
+>;
+
+const mapPositionToTransitionSide = {
+  bottom: "top",
+  left: "right",
+  right: "left",
+} as const;
+
+export const DrawerContent = forwardRef<HTMLDivElement, DrawerContentProps>(
+  ({ children, position = "right", ...props }, ref) => {
+    const { open } = useDrawerContext("DrawerContent");
+
+    return (
+      <AnimatePresence>
+        {open && (
+          <RadixDrawer.Portal forceMount>
+            <Transition>
+              <Backdrop asChild>
+                <RadixDrawer.Overlay />
+              </Backdrop>
+            </Transition>
+
+            <Transition
+              data-side={
+                mapPositionToTransitionSide[
+                  position as keyof typeof mapPositionToTransitionSide
+                ]
+              }
+              type="fade"
+            >
+              <Paper
```

Comment:
Vai, when there's no focusable item in the content, opening the drawer has a black outline for edge / blue outline for chrome.

Reply:
That is expected since dialog will focus trap you inside drawer

but you’ll always have something focusable in the content - at the very least you’ll have a close button. So we should be good.

Reply:
Got it vai.